main.py

```diff
@@ -88,7 +88,7 @@ def shutdown_session(exception=None):
     get_db().remove()
 
 
-@app.route('/v1/auth', methods=['POST'])
```

Comment:
Keeping versions is important to know what we have added, it doesn't only mean we had to fix something, versions represent a change, and that could be a new feature that we didn't think about before, that and, paths won't work without the version because of the way our files are set up.